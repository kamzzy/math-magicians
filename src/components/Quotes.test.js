import React from 'react';
import { render, screen } from '@testing-library/react';
import Quotes from './Quotes';

describe('render Quotes page', () => {
  test('render p element content', () => {
    render(<Quotes />);
    const p = screen.findAllByText('Mathematics');
    expect(p).toMatchSnapshot();
  });
});
