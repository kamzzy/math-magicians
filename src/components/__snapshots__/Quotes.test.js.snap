// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Quotes page render p element content 1`] = `Promise {}`;
